Makefile

```diff
@@ -241,6 +242,14 @@ can-build-pr:
 i18n-extract: ## Extract strings for translation from the source code
 	npm run mmjstool -- i18n extract-mobile
 
+i18n-extract-ci:
+	mkdir -p tmp
+	cp assets/base/i18n/en.json tmp/en.json
+	mkdir -p tmp/fake-webapp-dir/i18n/
+	echo '{}' > tmp/fake-webapp-dir/i18n/en.json
+	npm run mmjstool -- i18n extract-mobile --webapp-dir tmp/fake-webapp-dir --mobile-dir .
+	diff tmp/en.json assets/base/i18n/en.json
```

Comment:
lets remove the `tmp` dir after this step and we should be good to go

Reply:
Done, thanks.
I didn't remove the `tmp/` in .gitignore. I guess it's fine, but let me know otherwise.

Reply:
Yes leave it in .gitignore, that’s fine